public/js/icinga/behavior/modal.js

```diff
@@ -29,6 +29,11 @@
 
     Modal.prototype = new Icinga.EventListener();
 
+    /**
+     * Element that has opened the modal
+     */
+    let modalOpener;
```

Comment:
That's not an instance property. Also, `let` is ECMA2015 only and we still have to support IE.